tests/functional/cephx_keyrotation/test_cephx_keyrotation_mismatch_alert.py

```
@@ -0,0 +1,1042 @@
+                continue
+            actual = _metric_value(sample)
+            if actual != expected:
+                mismatches[daemon] = actual
+        if not mismatches:
+            log.info("Mismatch metric values match expected: %s", expected_by_daemon)
+            return by_daemon
+        log.info("Mismatch metric not yet ready: %s", mismatches)
+
+    raise UnexpectedBehaviour(
+        f"{MISMATCH_METRIC} did not reach {expected_by_daemon} within {timeout}s"
+    )
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)